src/Microsoft.Data.Analysis/ArrowStringDataFrameColumn.cs

```diff
@@ -147,10 +147,10 @@ public IEnumerable<ReadOnlyMemory<byte>> GetReadOnlyNullBitMapBuffers()
         }
 
         /// <summary>
-        /// Returns an enumerable of immutable ReadOnlyMemory<int> representing offsets into its corresponding Data buffer.
+        /// Returns an enumerable of immutable <see cref="ReadOnlyMemory{Int32}"/> representing offsets into its corresponding Data buffer.
```

Comment:
I've seen "enumeration" used more than "enumerable":
```suggestion
        /// Returns an enumeration of immutable <see cref="ReadOnlyMemory{Int32}"/> representing offsets into its corresponding Data buffer.
```